### PR TITLE
CSSglowingbutton/zareenf01

### PR DESCRIPTION
**Pull request by zareenf01**
I created a glowing button on hover with CSS

### CODE REVIEW

**Comment by avincodes** [comment]
Can you please add a readme file showing an example? Thanks!

**Comment by zareenf01** [comment]
Adding a preview

https://user-images.githubusercontent.com/109251356/194756267-1d464c2f-86bf-48af-9652-c0f26085954f.mp4

**Comment by avincodes** [comment]
Could you add this to a readme file in the folder? Thanks!

**Comment by avincodes** [comment]
No @zareenf01 the structure has to be 

[Snippet Folder] --> [[Code] + [ReadMe]]

**Comment by zareenf01** [comment]
Please check now

File: CSSglowingbutton/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# CSS GLOWING BUTTON
+
```
Comment (avincodes) [inline CSSglowingbutton/README.md:2]: Please add the preview here!

**Comment by avincodes** [comment]
Commented!

**Comment by avincodes** [comment]
Perfect! Thank you @zareenf01... Don't forget the star the repo!

**Pull request merged**